packages/lib/delegationCredential/server.ts

```diff
@@ -211,13 +215,32 @@ export async function getAllDelegationCredentialsForUser({ user }: { user: { ema
   return delegationCredentials;
```

Comment:
This log.debug seems very dangerous, is delegationCredentials good to log (contrary to its name? my instinct says this is bad)